tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -77,7 +77,7 @@ int es = PAPI_NULL; //event set
 
 size_t NB_ITER = CONST_NB_ITER;
 size_t M = WG_MULT0*J_CL_DEVICE_CONST_WORK_GROUP_SIZE0+JACOBI_DELTA;
-size_t N = WG_MULT1*J_CL_DEVICE_CONST_WORK_GROUP_SIZE1+JACOBI_DELTA;
+size_t K = WG_MULT1*J_CL_DEVICE_CONST_WORK_GROUP_SIZE1+JACOBI_DELTA;
```

Comment:
On a more general basis, shouldn't we prefix library identifiers with something to separate them from user code a bit better? Prefixing with a simple underscore is reserved for CRT and STL implementations (for this very reason), but something uniform and unique could be made coherent across the headers.